<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaneData</class>
 <widget class="QWidget" name="PaneData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1522</width>
    <height>1011</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1504</width>
        <height>935</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Data</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="buttonRun">
           <property name="text">
            <string>Run</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QListView" name="listViewTemplates">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTableView" name="tableViewParams">
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
          </widget>
          <widget class="QTableView" name="tableViewPairs">
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
          </widget>
          <widget class="QListView" name="listViewIndicators">
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>From</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateTimeEdit" name="dateTimeEditFrom"/>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>To</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateTimeEdit" name="dateTimeEditTo"/>
             </item>
             <item>
              <widget class="QListWidget" name="listChartType">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="flow">
                <enum>QListView::LeftToRight</enum>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
               <item>
                <property name="text">
                 <string>Normal</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Log</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>6</number>
             </property>
             <item>
              <widget class="QStackedWidget" name="stackedWidgetCharts">
               <property name="currentIndex">
                <number>1</number>
               </property>
               <widget class="QWidget" name="page_3">
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QChartView" name="chartViewPrice"/>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="page_4">
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QChartView" name="chartViewPriceLog"/>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
             <item>
              <widget class="QChartView" name="chartViewVolume"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1504</width>
        <height>935</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Log</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextEdit" name="textEditLog"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>listChartType</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidgetCharts</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1001</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>1495</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
